runtime/doc/usr_04.txt

```diff
@@ -122,6 +122,7 @@ single-letter command:
 	C  stands for  c$  (change to end of the line)
 	s  stands for  cl  (change one character)
 	S  stands for  cc  (change a whole line)
+	Y  stands for  y$  (yank to end of the line)
```

Comment:
I think there is a reason why Vim doesn't mention `Y` here: this is in the "Changing text" section, and yanking is in the "Moving text" section below.